components/contribute-cards/Contribute.js

```diff
@@ -180,14 +193,17 @@ const ContributeCard = ({
   stats,
   hideContributors,
   image,
+  color,
   disableCTA,
   ...props
 }) => {
   const totalContributors = (stats && stats.all) || (contributors && contributors.length) || 0;
+  const customTheme = color && getTheme(color);
```

Comment:
`getTheme` is pretty expensive, we do not want to re-generate a full theme for each contribution card. I suggest using the provided color directly, we can always introduce a `normalizePrimaryColor` helper later if we get poor results.